src/components/common/topicsFilter/index.tsx

```diff
@@ -235,7 +235,7 @@ const CreateTopic = ({ onCreateCamp = () => {} }) => {
                 Canonizer Algorithm:
               </Title>
               <Popover content="Algorithm Information" placement="top">
-                <NextLink href="/topic/53-Canonizer-Algorithms/1-Agreement">
+                <NextLink href="/topic/132-Help/1-Agreement">
```

Comment:
Please move this URL to config. So we can manage it directly from the config file

Reply:
@akuks Moved URL to constants file